packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -95,6 +96,13 @@ class Scrollbar extends StatelessWidget {
   /// {@macro flutter.widgets.Scrollbar.isAlwaysShown}
   final bool? isAlwaysShown;
 
+  /// Controls if the track will show in all descendant [Scrollbar] widgets.
+  ///
+  /// If this property is null, then [ScrollbarThemeData.trackVisibility] of
+  /// [ThemeData.scrollbarTheme] is used. If that is also null, the track will
+  /// not show by default.
+  final MaterialStateProperty<bool?>? trackVisibility;
```

Comment:
Can you document how this relates (or doesn't) to `showTrackOnHover`? I know that we are going to follow up with deprecating the later, but in the meantime it may be difficult to understand which one I should use and how they affect each other, or if one overrides the other.